src/components/SharableLink.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+	<span class="querybuilder__sharable-link">
+		{{ $i18n('query-builder-sharable-link-text') }}
+		<Popover
+			:reactToHover="false"
+			position="end"
+		>
+			<template v-slot:target>
+				<Button
+					variant="normal"
+					type="neutral"
+					iconOnly
+					@click.native="copyTextToClipboard"
+					aria-label="Sharable link">
+					<Icon type="link" size="medium" />
+				</Button>
+			</template>
+			<template v-slot:default>
+				{{ $i18n('query-builder-sharable-link-message') }}
+			</template>
+		</Popover>
+	</span>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { Icon, Button, Popover } from '@wmde/wikit-vue-components';
+import QuerySerializer from '@/serialization/QuerySerializer';
+
+export default Vue.extend( {
+	name: 'SharableLink',
+	components: {
+		Icon,
+		Button,
+		Popover,
+	},
+	data: function () {
+		return {
+			href: '' as string,
+		};
+	},
+	methods: {
+		async copyTextToClipboard(): Promise<void> {
+			const querySerializer = new QuerySerializer();
+			const serializedQuery = querySerializer.serialize( this.$store.state );
+			const current = new URL( window.location.href );
+			current.searchParams.set( 'query', serializedQuery );
+			this.href = current.href;
+			try {
+				await navigator.clipboard.writeText( current.href );
+			} catch ( err ) {
+				// pass
```

Comment:
what does 'pass' mean. is something supposed to happen here ?

Reply:
good point. let me just remove it.

Reply:
Removing the catch makes some tests to warn, I changed it to a better comment why it should ignore failure.